invenio_app_rdm/records_ui/previewer/iiif_simple.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2022 data-futures.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Simple IIIF image preview."""
+
+from os.path import splitext
+
+from flask import current_app, render_template
+
+previewable_extensions = ['gif', 'jpg', 'jpeg', 'jp2', 'png', 'tif', 'tiff']
```

Comment:
Minor: I see that this variable is not used and also matches the `supported_extensions` variable inside `can_preview` below), so it can be removed.

Reply:
I thought this too, but its common to all invenio_previewers. On closer inspection the `previewable_extensions` property is used externally: 

https://github.com/inveniosoftware/invenio-previewer/blob/08e6c6ded18cea8d197ee7890183b71d71e849a9/invenio_previewer/views.py#L75

So I left it in.

I agree its redundant - but I've tried to follow the pattern of the other previewers - the need to include the '.' is also messy - I considered setting `supported_extensions = ['.' + ext for ext in previewable_extensions]` - to at least remove the chance of the 2 lists not agreeing, what do you think?

Reply:
You're right about the `previewable_extensions`, just realized too there's this "implicit" API requirement... It's a bit unfortunate since there's nothing to help validate/enforce this. I feel it would be much better to have a class-based API for implementing previewers so that one can just implement a couple of properties/methods and have some guidance and validation.

I think one way to move forward in order to have also the configuration option available, is to try something like:

```python
from werkzeug.local import LocalProxy
from flask import current_app

previewable_extensions = LocalProxy(
    lambda: current_app.config['IIIF_SIMPLE_PREVIEWER_SUPPORTED_EXTENSIONS'])
```

Agree also on the `supported_extensions` inconsistency with the `.` prefixing, what you propose for generating the list sounds good.

Reply:
Done, will do some more testing and possibly tweak default settings now in config.py